function/Dockerfile

```diff
@@ -19,9 +22,9 @@ RUN python3 ./pre_download.py
 
 COPY *.txt ./
 
-
-
 ENV write_debug="true"
 ENV fprocess="python3 index.py"
 
+USER 10001
```

Comment:
Can you instead add a new user and use that user like we do in the standard templates?

see openfaas/templates